fem/datacollection.hpp

```diff
@@ -445,6 +446,10 @@ class VisItDataCollection : public DataCollection
    /// Add a grid function to the collection and update the root file
    virtual void RegisterField(const std::string& field_name, GridFunction *gf);
 
+   /// Add a quadrature function to the collection and update the root file
+   virtual void RegisterQField(const std::string& q_field_name,
```

Comment:
Another comment should be added that this currently is not usable within Visit to visualize quadrature data.

Based on the below unit test, if you do try to within Visit it fails with an error from MFEM saying: ```MFEM abort: input stream is not a FiniteElementSpace!```

Reply:
Will make a statement that VisIt 3.12 does not work. A patch has been sent to VisIt developers.